### PR TITLE
WINDUPRULE-287: Statistics Rules: JMS Messaging resources

### PR DESCRIPTION
**Pull request by jsight**
Depends on: https://github.com/windup/windup/pull/1169

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -230,8 +230,8 @@
                     <validationSets>
                         <validationSet>
                             <includes>
-                                <include>**/*.windup.xml</include>
-                                <include>**/*.windup.test.xml</include>
+                                <include>**/*.awindup.xml</include>
+                                <include>**/*.awindup.test.xml</include>
```
Comment (mareknovotny) [inline pom.xml]: what's is the "a" letter in wildcard inclusion?

Comment (jsight) [reply pom.xml]: A hack since I had not written the schema yet. I'll work on that now. :)

Comment (mareknovotny) [reply pom.xml]: ok, :)

Comment (jsight) [reply pom.xml]: Fixed

**Comment by mareknovotny** [comment]
retest

**Comment by mareknovotny** [comment]
retest

**Comment by mareknovotny** [comment]
retest

**Comment by jsight** [comment]
retest

**Pull request merged**